bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -123,13 +123,30 @@ TEST_F(InstanceAdminIntegrationTest, ListClustersTest) {
   if (UsingCloudBigtableEmulator()) {
     return;
   }
+  std::string id = "list-clusters-test";
+  bigtable::InstanceId instance_id(id);
+  bigtable::DisplayName display_name(id);
+  std::vector<std::pair<std::string, bigtable::ClusterConfig>> clusters_config;
+  clusters_config.push_back(std::make_pair(
+      id + "-cluster1", bigtable::ClusterConfig("us-central1-f", 0,
+                                                bigtable::ClusterConfig::HDD)));
+  auto instance_config =
+      bigtable::InstanceConfig(instance_id, display_name, clusters_config)
+          .set_type(bigtable::InstanceConfig::DEVELOPMENT);
+  auto instance_details =
+      instance_admin_->CreateInstance(instance_config).get();
+
+  // Create clusters in an instance
+  // TODO(#422) - Implement InstanceAdmin::CreateCluster
 
   // TODO(#418) - create an instance and test that its cluster is returned here.
-  auto clusters = instance_admin_->ListClusters();
+  auto clusters = instance_admin_->ListClusters(id);
```

Comment:
This test would pass even if `ListClusters()` returns an empty vector.  Please check that the collection is not empty.